modules/fedimint-wallet-common/Cargo.toml

```diff
@@ -22,7 +22,8 @@ bitcoin = { version = "0.29.2", features = [ "rand", "serde"] }
 erased-serde = "0.3"
 fedimint-core = { version = "0.3.0-alpha", path = "../../fedimint-core" }
 futures = "0.3"
-miniscript = { version = "9.0.2", features = [ "compiler", "serde" ] }
+miniscript = { version = "10.0.0", features = [ "compiler", "serde" ] }
+miniscript9 = { package = "miniscript", version = "9.0.2" }
```

Comment:
Why does `wallet-common` need both versions of `miniscript`?

Reply:
Because this piece of code returns a miniscript v9 error:

https://github.com/fedimint/fedimint/blob/28ecd67ca0c7279c9d2d5d00f34fe20fc77e3062/modules/fedimint-wallet-server/src/lib.rs#L909-L911

We could add a function to `bitcoin_migration.rs` to convert this, but the error type has [40 variants](https://docs.rs/miniscript/9.0.1/miniscript/enum.Error.html) and I don't know that we really use it for much anyway. I figure it's easier to wait until we upgrade to bitcoin v30 to swap this out.